packages/nvidia_nat_vanna/src/nat/plugins/vanna/execute_db_query.py

```
@@ -0,0 +1,252 @@
+
+    # Query configuration
+    max_rows: int = Field(default=100, description="Maximum rows to return")
+
+
+@register_function(
+    config_type=ExecuteDBQueryConfig,
+    framework_wrappers=[LLMFrameworkEnum.LANGCHAIN],
+)
+async def execute_db_query(
+    config: ExecuteDBQueryConfig,
+    builder: Builder,
```

Ruff (0.14.2)

Unused function argument: `builder`

(ARG001)